src/main/game/HumanPlayer.java

```diff
@@ -3,23 +3,24 @@
 public class HumanPlayer implements Player {
 
     private final UI ui;
-    private final String mark;
+    private final Mark mark;
 
-    public HumanPlayer(UI ui, String mark) {
+    public HumanPlayer(UI ui, Mark mark) {
         this.ui = ui;
         this.mark = mark;
     }
 
     public void playMove(Board board) {
-        String playerMove = getMove(board);
+        int playerMove = getMove(board);
```

Comment:
Consistent naming. In your Computer class you called this `move` so I would drop the `get` from here too.